src/main/java/com/github/manolo8/darkbot/gui/titlebar/ConfigPicker.java

```diff
@@ -40,6 +42,19 @@ public ConfigPicker() {
         list.addMouseListener(renderer);
         list.addMouseMotionListener(renderer);
 
+        addPopupMenuListener(new PopupMenuListener() {
+            @Override
+            public void popupMenuWillBecomeVisible(PopupMenuEvent e) {
+                setup(main);
```

Comment:
setup is a function that may be potentially blocking, as it's listing the files in a directory, and that may take an indeterminate amount of time (eg: imagine you run off of a remote disk). That's an issue as the GUI may freeze every time you use the dropdown.

Since the issue is likely to never actually be of much affect i'm going to ignore it, but it'd be nice to have it fixed so that configManager#getAvailableConfigs is called using a swing worker, and then, when done, on the swing thread, you can just re-create the list. That way you open the drop down, and as soon as the update is ready you'll see the new entries appear in the list